isolated-buttons/src/app.js

```diff
@@ -0,0 +1,16 @@
+import {Subject} from 'rx'
+import {run} from '@cycle/core'
+import {makeDOMDriver} from 'cycle-snabbdom'
+
+import {createButton} from './Button'
+
+function main(sources) {
+  const Button = createButton({id: '0', removable: false})
```

Comment:
I got a bit confused here, maybe if it would be named `createButtonComponent` I wouldn't be confused.